### PR TITLE
#GOV-437 Global parameters to re-use in multiple locations

### PR DESCRIPTION
**Pull request by somanath21**
(https://mifosforge.jira.com/browse/GOV-437) Description

* Added a Global parameter for the below variables
1. SPRING_PROFILES_ACTIVE
2. ImagePullPolicy
3. LOGGING_LEVEL_ROOT
4. LOGGING_PATTERN_CONSOLE
5. tenants
6. DFSPIDS     

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Comment by fynmanoj** [comment]
rebase this PR

**Review by fynmanoj — APPROVED** [review]

**Comment by fynmanoj** [comment]
The checks should be green to merge it

**Review by fynmanoj — CHANGES_REQUESTED** [review]
please update the global parameters in ID account mapper

**Comment by fynmanoj** [comment]
please rebase

File: helm/g2p-sandbox/Chart.yaml [hunk 1]
```diff
@@ -7,5 +7,5 @@ appVersion: 0.0.0
 
 dependencies:
 - name: ph-ee-engine
-  repository: https://fynarfin.io/images/ph-ee-engine-0.0.0-SNAPSHOT
-  version: 0.0.0-SNAPSHOT
+  repository: https://fynarfin.io/images/ph-ee-engine-1.8.0-SNAPSHOT
+  version: 1.8.0-SNAPSHOT
```
[the remark this answers is not in the document]
Comment (somanath21) [reply helm/g2p-sandbox/Chart.yaml]: Done

**Comment by fynmanoj** [comment]
Identity and Voucher services to be included

**Pull request merged**